### PR TITLE
Bjorncs/docker build cleanup

### PR DESCRIPTION
**Pull request by bjorncs**
FYI @aressem

### CODE REVIEW

File: dist/vespa.spec [hunk 1]
```diff
@@ -67,8 +67,8 @@ Vespa - The open big data serving engine
 
 %build
 source /opt/rh/devtoolset-6/enable || true
-sh bootstrap.sh
-mvn install -DskipTests -Dmaven.javadoc.skip=true
+sh bootstrap.sh java
+mvn -nsu -T %{_smp_mflags} install -DskipTests -Dmaven.javadoc.skip=true
```
Comment (aressem) [inline dist/vespa.spec]: Nice

File: docker/build-vespa.sh [hunk 2]
```diff
@@ -11,8 +11,7 @@ DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
 cd $DIR
 
 VESPA_VERSION=$1
-DOCKER_IMAGE="vespabuild"
+DOCKER_IMAGE="vespaengine/vespa-dev:latest"
```
Comment (aressem) [inline docker/build-vespa.sh]: Suggest doing a docker pull vespaengine/vespa-dev to make sure you have the latest. If already have the image with this tag, docker will not automatically pull it.

File: docker/enter-build-container.sh [hunk 3]
```diff
@@ -10,8 +10,7 @@ fi
 DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
 cd $DIR
 
-DOCKER_IMAGE="vespabuild"
+DOCKER_IMAGE="vespaengine/vespa-dev:latest"
```
Comment (aressem) [inline docker/enter-build-container.sh]: Same as above. Do a docker pull.

Comment (bjorncs) [inline docker/enter-build-container.sh]: Fixed

File: docker/build-vespa.sh (hunk 2 above)
Comment (bjorncs) [inline docker/build-vespa.sh]: Fixed

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**